### PR TITLE
Add deno image

### PR DESCRIPTION
**Pull request by hardy613**
## Description
Adding [`deno`](https://deno.land)  image with a tag `v1.0` for Ubuntu `18.4`

## Steps
1. build images
2. `docker run -it aghost7/deno-dev:bionic` 
3. `deno` or `deno re<tab>` to test tab completion

Screenshot from 2020-05-18 17-32-42

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -33,7 +33,11 @@
     {'name': 'nodejs-dev', 'tag': 'bionic-dubnium',
         'path': 'nodejs-dev/dubnium', 'args': {'BASE_TAG': 'bionic'}},
     {'name': 'c-dev', 'tag': 'bionic'},
-    {'name': 'devops', 'tag': 'bionic'}
+    {'name': 'devops', 'tag': 'bionic'},
+    {'name': 'deno-dev-base', 'path': 'deno-dev/base', 'tag': 'bionic',
```
Comment (AGhost-7) [inline build.py:37]: We don't need the base image since we only have one public tag.

File: images/deno-dev/base/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+COPY ./plugin.vim /tmp/plugin.vim
+
+COPY ./post-plugin.vim /tmp/post-plugin.vim
+
+ONBUILD COPY .deno-version $HOME/.deno-version
+
+RUN /tmp/build.sh && sudo rm /tmp/build.sh
+
+ENV DENO_INSTALL="/$HOME/.deno"
```
Comment (AGhost-7) [inline images/deno-dev/base/Dockerfile:21]: Extra slash.

File: build.py [hunk 3]
```diff
@@ -33,7 +33,9 @@
     {'name': 'nodejs-dev', 'tag': 'bionic-dubnium',
         'path': 'nodejs-dev/dubnium', 'args': {'BASE_TAG': 'bionic'}},
     {'name': 'c-dev', 'tag': 'bionic'},
-    {'name': 'devops', 'tag': 'bionic'}
+    {'name': 'devops', 'tag': 'bionic'},
+    {'name': 'deno-dev', 'tag': 'v1.0', 'path': 'deno-dev/v1.0',
```
Comment (AGhost-7) [inline build.py:37]: Please follow the same convention as the ruby and python tags.

Comment (hardy613) [reply build.py:37]: should I remove the paths then and just use `deno-dev`

File: images/deno-dev/post-plugin.vim [hunk 4]
```diff
@@ -0,0 +1,19 @@
+" Simple debugger integration.
+
+fu! g:SendDBBreak()
```
Comment (AGhost-7) [inline images/deno-dev/post-plugin.vim:3]: Is there even a cli-based debugger for deno?

Comment (hardy613) [reply images/deno-dev/post-plugin.vim:3]: yes, but I might have it set up wrong, thoughts?

https://deno.land/manual/tools/debugger

**Pull request merged**